gdal/frmts/hdf5/bagdataset.cpp

```diff
@@ -3344,8 +3344,8 @@ void BAGDataset::LoadMetadata()
                     fabs((dfURY - dfLLY) / dfResHeight - m_nLowResHeight) < 1e-2 )
                 {
                     // Found with https://data.ngdc.noaa.gov/platforms/ocean/nos/coast/H12001-H14000/H12525/BAG/H12525_MB_4m_MLLW_1of2.bag
-                    // https://github.com/OSGeo/gdal/issues/1643
-                    CPLError(CE_Failure, CPLE_AppDefined,"Ignoring BAG Upper Right Corner.");
+                    // to address issue https://github.com/OSGeo/gdal/issues/1643
+                    CPLError(CE_Warning, CPLE_AppDefined, "cornerPoints not consistent with resolution given in metadata");
```

Comment:
I'd suggest a more specific message for that case: "cornerPoints and resolution in metadata seem to imply a PixelCorner convention, whereas BAG mandates PixelCenter. GDAL will georeference the product assuming this non-standard PixelCorner convention"